opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/opennms-minion-graph.properties

```diff
@@ -116,8 +112,65 @@ report.OpenNMS.Minion.Syslogd.Kafka.Exchanges.command=--title="Syslog Messages S
  GPRINT:complete:MIN:" Min \\: %8.2lf %s" \
  GPRINT:complete:MAX:" Max \\: %8.2lf %s\\n" \
 
+###########################################
+## OpenNMS.Minion.Syslogd.JMS.ProcessingTime
+###########################################
+report.OpenNMS.Minion.Syslogd.JMS.ProcessingTime.name=Syslog Message Processing Time
+report.OpenNMS.Minion.Syslogd.JMS.ProcessingTime.columns=SlogListLastProc, SlogListMeanProc, SlogMarLastProc, SlogMarMeanProc, SlogJmsLastProc, SlogJmsMeanProc
+report.OpenNMS.Minion.Syslogd.JMS.ProcessingTime.type=interfaceSnmp
+report.OpenNMS.Minion.Syslogd.JMS.ProcessingTime.command=--title="Syslog Message Processing Time" \
+ DEF:listLast={rrd1}:SlogListLastProc:AVERAGE \
+ DEF:listMean={rrd2}:SlogListMeanProc:AVERAGE \
+ DEF:marLast={rrd3}:SlogMarLastProc:AVERAGE \
+ DEF:marMean={rrd4}:SlogMarMeanProc:AVERAGE \
+ DEF:mqLast={rrd5}:SlogJmsLastProc:AVERAGE \
+ DEF:mqMean={rrd6}:SlogJmsMeanProc:AVERAGE \
+ CDEF:listLastSec=listLast,1000,/ \
+ CDEF:marLastSec=marLast,1000,/ \
+ CDEF:mqLastSec=mqLast,1000,/ \
+ AREA:listLastSec#ad7fa8:"Receive       " \
+ GPRINT:listLastSec:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:listLastSec:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:listLastSec:MAX:" Max \\: %8.2lf %s\\n" \
+ STACK:marLastSec#729fcf:"Process       " \
+ GPRINT:marLastSec:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:marLastSec:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:marLastSec:MAX:" Max \\: %8.2lf %s\\n" \
+ STACK:mqLastSec#73d216:"Send via JMS  " \
+ GPRINT:mqLastSec:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:mqLastSec:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:mqLastSec:MAX:" Max \\: %8.2lf %s\\n" \
```

Comment:
No, and it does unexpectedly extend the line continuation. However, it makes it easier to cut-and-paste the lines to make additional line/area defs. You just have to make sure to always put a blank line afterwards.

I'll go and clean these up because after looking at all of the graph configs, only about 2% have this trailing slash. I'll extend the unit tests to detect this so that we're consistent.